uk.co.compucorp.civicrm.hrleaveandabsences/api/v3/LeavePeriodEntitlementLog.php

```diff
@@ -0,0 +1,53 @@
+<?php
+use CRM_HRLeaveAndAbsences_ExtensionUtil as E;
+
+/**
+ * LeavePeriodEntitlementLog.create API specification (optional)
+ * This is used for documentation and validation.
+ *
+ * @param array $spec description of fields supported by this API call
+ * @return void
+ * @see http://wiki.civicrm.org/confluence/display/CRMDOC/API+Architecture+Standards
+ */
+function _civicrm_api3_leave_period_entitlement_log_create_spec(&$spec) {
+  // $spec['some_parameter']['api.required'] = 1;
+}
+
+/**
+ * LeavePeriodEntitlementLog.create API
+ *
+ * @param array $params
+ *
+ * @return array API result descriptor
+ *
+ * @throws API_Exception
+ */
+function civicrm_api3_leave_period_entitlement_log_create($params) {
+  return _civicrm_api3_basic_create(_civicrm_api3_get_BAO(__FUNCTION__), $params);
+}
```

Comment:
Since people should not be able to create or update log entries using the API, I think it would be better to completely remove this endpoint